README.md

```diff
@@ -392,16 +380,9 @@ addEventListener("click", () => score++)!
 print(await next score)! //6 (when you click)
 ```
 
-Additionally, the `current` keyword lets you see into the present.
```

Comment:
Don't remove the `current` keyword

Reply:
It's not redundant because of this case:
```js
const const val = test !
var var value = tests !

print current value = val // false
print value = val // true
```